script.js

```diff
@@ -1,4 +1,275 @@
+
+/* 
+Rules:
+
+1) There are 2 players and players take turns.
+
+2) When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+
+3) The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+
+4) After both players have rolled and chosen dice order, the player with the higher combined number wins.
+*/
+
+//Create global variables that stores whether the players have rolled
+var hasPlayer1Rolled = 0;
+var hasPlayer2Rolled = 0;
+
+//Create a global variable that is an array that will store the players' rolled numbers. It is empty at first but when the numbers are pushed in it will be in the following format ideally: [player1Dice1, player1Dice2] [player2Dice1, player2Dice2]
+var player1DiceRolls = [];
+var player2DiceRolls = [];
+
+//Create global variables for the players' concatenated numbers
+var player1ConcatenatedNumber = 0;
+var player2ConcatenatedNumber = 0;
+
+//Create a global variable to store the reset mode
+var timeToReset = 0;
+
+//Create a function that capitalizes the first letter of a string
+var capitalizer = function (userInput) {
+  return userInput[0].toUpperCase() + userInput.substring(1);
+};
+
+//Create the dice roll function
+var randomDiceNumberGenerator = function () {
+  return Math.floor(Math.random() * 6) + 1;
+};
+
+//Create a function to roll the players' dices, depending on whether its player 1 turn or player 2 turn
+var rollTheDice = function () {
+  //If Player 1 has not rolled (hasPlayer1Rolled = 0), roll the dice twice and push the numbers to the player1DiceRolls array. If Player 1 has rolled (hasPlayer1Rolled = 1), then check if player 2 has not rolled (hasPlayer2Rolled = 0), roll the dice twice and push the numbers to the player2DiceRolls array.
+  if (hasPlayer1Rolled == 0) {
+    //Push two dice rolls into Player 1 Dice Roll's array, player1DiceRolls, using a for loop
+    for (var counter = 0; counter < 2; counter += 1) {
+      player1DiceRolls.push(randomDiceNumberGenerator());
+    }
+    console.log(
+      "Player 1 Dice 1: ",
+      player1DiceRolls[0],
+      "Player 1 Dice 2: ",
+      player1DiceRolls[1]
+    );
+
+    //Change the global variable hasPlayer1Rolled to 1 to show that Player 1 has rolled
+    hasPlayer1Rolled = 1;
+  } else if (hasPlayer1Rolled == 1 && hasPlayer2Rolled == 0) {
+    //Push two dice rolls into Player 2 Dice Roll's array, player1DiceRolls, using a for loop
+    for (var counter = 0; counter < 2; counter += 1) {
+      player2DiceRolls.push(randomDiceNumberGenerator());
+    }
+    console.log(
+      "Player 2 Dice 1: ",
+      player2DiceRolls[0],
+      "Player 2 Dice 2: ",
+      player2DiceRolls[1]
+    );
+    //Change the global variable hasPlayer1Rolled to 1 to show that Player 1 has rolled
+    hasPlayer2Rolled = 1;
+  }
+};
```

Comment:
this function is ok and nice use of loops to push your dice rolls in. since we are only dealing with 2 rolls right now, maybe a loop is unnecessary and you can just do something like `player1DiceRolls = [randomDiceNumberGenerator(), randomDiceNumberGenerator()]` but definitely use loops if we are dealing with more dice rolls! 

Feels like this function can be merged together with `statementForDiceRolled` where the return statements are incorporated into `rolltheDice` the relevant places. Lines 199-200 can then just be `return rollTheDice()`